notion/interfaces.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC, abstractmethod
+
+from apis.interfaces import IAPIClient
+
+
+class INotionDataParser(ABC):
+    @abstractmethod
+    def parse(self, data: dict):
+        pass
+
+    @abstractmethod
+    def _format_text(self, text: str, value_type: str):
```

Comment:
Should private methods be defined in abstract classes, let's check on whether it is required